_posts/2018-10-03-write-better.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title:  "Write Better!"
+date:   2018-10-03 10:01:10 +0100
+categories: community contributions
+---
```

Comment:
Worth including the author tag in here? There's an issue to add authors to posts. Perhaps an author 'link' too?

Reply:
Good idea! I've just added an author tag to the front matter.